ArduCopter/AP_Arming.cpp

```diff
@@ -260,7 +260,7 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
                 uint16_t terr_pending, terr_loaded;
                 copter.terrain.get_statistics(terr_pending, terr_loaded);
                 if (terr_pending != 0) {
-                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, waiting for terrain data");
```

Comment:
we should leave this one as it was..

Otherwise the change looks good, thanks!